CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## [1.9.11] - 2019-11-01
```

Comment:
This line should be: 
`## Unreleased`

Reply:
@waldman's comment was marked as resolved even though the code shows otherwise. I'm going to unresolve until there's a change to the code.

Reply:
I see.  Is this because you don't give subver till merge to master?

Reply:
That's correct. It's [stated on step 8 of GitFlow](https://datasift.github.io/gitflow/GitFlowForGitHub.html#8-creating-releases):

> Once you’ve created the release branch, **remember to update the version number in your code** (in the pom.xml, Makefile, build.xml or wherever it is stored).

In this case "wherever it is stored" is the `CHANGELOG`. 😄

Reply:
I have never tried using the gitflow toolset.  Always done gitflow manually.  Make life much easier?

Reply:
[The git hubflow tool](https://datasift.github.io/gitflow/TheHubFlowTools.html#installing-hubflow) only helps with automatically managing the local and remote branches. It won't write any code for you. 

I'm not sure if that last bit was an actual question. But yes, it definitely makes life easier, especially when it's time to merge and tag your releases.